modules/cloudsql-instance/variables.tf

```diff
@@ -245,6 +239,22 @@ variable "root_password" {
   default     = null
 }
 
+variable "ssl" {
+  description = "Setting to enable SSL, set config and certificates."
+  type = object({
+    require_ssl = optional(bool, false)
+    # More details @ https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/sql_database_instance#ssl_mode
+    ssl_mode            = optional(string)
+    client_certificates = optional(list(string))
+
+  })
+  default = null
```

Comment:
```
default = {}
nullable = false
```

then you can remove the `try`s from the validation rules.

Reply:
can you add `nullable = false`?

Reply:
it should be there, please check the new version

Reply:
thanks :)